src/ActionsImporter/Commands/Configure.cs

```diff
@@ -1,20 +1,32 @@
 ﻿using System.CommandLine;
-using System.CommandLine.NamingConventionBinder;
+using ActionsImporter.Handlers;
 
 namespace ActionsImporter.Commands;
 
 public class Configure : BaseCommand
 {
+    private readonly string[] _args;
     protected override string Name => "configure";
     protected override string Description => "Start an interactive prompt to configure credentials used to authenticate with your CI server(s).";
 
+    public static readonly Option<bool> OptionalFeaturesOption = new(new[] { "--optional-features" })
+    {
+        Description = "Configure features to customize your usage of GitHub Actions Importer"
+    };
+
+    public Configure(string[] args)
+    {
+        _args = args;
```

Comment:
I'm happy to change this, but it's worth noting that this pattern was copied from our other commands which don't explictly use a `this` reference (e.g. https://github.com/github/gh-actions-importer/blob/9c4395fb1015b5945ab5bc68e80f31f3b0aa333f/src/ActionsImporter/Commands/DryRun.cs#L9-L12)

So this would make this class look different. What do you think?